eks/ng/nodes.go

```diff
@@ -472,8 +468,14 @@ func (ts *tester) createASGs() error {
 				tg.UserData += fmt.Sprintf(` %s`, cur.KubeletExtraArgs)
 			}
 			tg.UserData += "'"
+			if cur.BootstrapArgs != "" {
+				ts.cfg.Logger.Info("adding further additional bootstrap arguments to user data",
+					zap.String("bootstrap-args", cur.BootstrapArgs),
+				)
+				tg.UserData += fmt.Sprintf(` %s`, cur.BootstrapArgs)
+			}
 			tg.UserData += "\n"
-			tg.UserData += `              /opt/aws/bin/cfn-signal --exit-code $? --stack ${AWS::StackName} --resource ASG --region ${AWS::Region}`
+			tg.UserData += `              /opt/aws/bin/cfn-signal --exit-code $? --stack ${AWS::StackName} --resource ASG --region ${AWS::Region} --url='https://cloudformation.${AWS::Region}.${AWS::URLSuffix}' --role='${RoleName}' \`
```

Comment:
Do we need ` \` at the end of the line? Seems like this would be the last command in user data?